patternfly-a11y.config.js

```diff
@@ -10,9 +10,22 @@ const urls = Object.keys(fullscreenRoutes)
       return key;
     } else {
       const path = fullscreenRoutes[key].path
-      if (path.match(/\/demos\/.*\/html-demos$/g)) {
+      if (path.match(/\/patterns\/.*\/html-demos$/g)) {
         return path.replace(/\/html-demos$/, '');
       } else {
+        if (
```

Comment:
Do we need to add context selector here? BTW - not sure if this is an issue but the demo page shows up first now, maybe that's intentional since the demos are still valid - they would just ideally be built using menu/menu toggle. Since core isn't going to have a context selector component, I don't know if the context selector demos belong there though. Or maybe the nav link needs a "demo" badge, like password strength/generator?